src/mpid/common/thread/mpidu_thread_fallback.h

```diff
@@ -95,8 +95,62 @@ typedef MPL_thread_func_t MPIDU_Thread_func_t;
 - _context - A context (typically an object) of the critical section
 
 M*/
+
+/*M MPIDU_THREAD_CS_TRYENTER - Try enter a named critical section
+
+  Input Parameters:
++ _name - name of the critical section
+- _context - A context (typically an object) of the critical section
+- cs_acqt  - A flag that indicades whether the critical section was acquired
```

Comment:
`cs_acqt` -> `_cs_acq`, `indicades` -> `indicates`

Reply:
fixed (and two more errors where `cs_req` was used for `cs_acq`)